src/harness/fake_sas.py

```diff
@@ -236,7 +236,9 @@ def do_POST(self):
                        '/admin/injectdata/sas_impl',
                        '/admin/injectdata/esc_sensor',
                        '/admin/trigger/meas_report_in_registration_response',
-                       '/admin/trigger/meas_report_in_heartbeat_response'):
+                       '/admin/trigger/meas_report_in_heartbeat_response',
+                       '/admin/trigger/create_full_activity_dump',
+                       '/v1.0/dump'):
```

Comment:
Since /v1.0/dump needs to return some data in the response, probably we need to have a slightly more sophisticated response from the fake SAS.